assets/stylesheets/custom_messages.css

```diff
@@ -36,6 +36,10 @@
   color: white;
 }
 
-p.toggle-enable {
+div.message-customize-menu {
   text-align: right;
+}
+div.message-customize-menu p {
+  color: gray;
+  font-size: 5px;
```

Comment:
Is the font-size too small?

Reply:
I changed to 5px.

Reply:
I think it ’s good.
However, I think the p tag margin(top, bottom) is wide. Probably the default style value of the browser.

Reply:
I think it ’s good.
However, I think the p tag margin(top, bottom) is wide. Probably the default style value of the browser.